03 Design/OCs/OCSetCustomerZIPCode.md

```diff
@@ -0,0 +1,25 @@
+#### Operation: 
+
+setCustomerZIPCode (customerZIPCode) 
+
+#### Cross reference: 
+
+Use case 1: Send Request 
+
+#### Preconditions: 
+
+Consultant consultant exists. 
+
+Customer customer exists. 
+
+consultant specifies customer.
+
+customerZIPCode exists.
+
+customerZIPCode is numeric.
+
+customerZIPCode contains 4 characters.
```

Comment:
I would prefer "digits" to "characters" since we've already specified that its a numeric value